src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+
+const ChatLog = (props) => {
+    const {entries} = props;
```

Comment:
We can combine both of these lines by using object restructuring:

```suggestion
const ChatLog = ({entries, updateMessage}) => {
```